content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -17,29 +17,119 @@ example application for this task.
 
 ## Before you begin
 
+Istio provides a choice of backend tracing providers.
+
+<details><summary>Jaeger or Zipkin</summary>
+
 *   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
 
     Either use the `istio-demo.yaml` or `istio-demo-auth.yaml` template, which includes tracing support, or
-    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option.
+    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option and optionally
+    selecting the required tracing provider using `--set tracing.provider=<provider>`. Currently supported
+    providers are `jaeger` (the default) and `zipkin`.
+
+</details>
+
+<details><summary>LightStep</summary>
```

Comment:
The structure of this content ins confusing, constantly switching between tools. I suggest you create three files. 
1. An overview file named _index.md in the existing folder. This file's description field has the first paragraph of this document as its value.
2. A folder for each tool containing and index.md with the task content for the tool. For example, `/distributed-tracing/zipkin/index.md.
3. The tool specific index.md files contain all the steps a reader needs to complete to achieve distributed tracing with that tool.